lms/envs/common.py

```diff
@@ -2050,3 +2050,25 @@
 SEARCH_ENGINE = None
 # Use the LMS specific result processor
 SEARCH_RESULT_PROCESSOR = "lms.lib.courseware_search.lms_result_processor.LmsSearchResultProcessor"
+
+######################## Leaner profile configuration ###########################
+
+# Default visibility level for accounts without a specified value
+# The value is one of: 'global', 'edx_users', 'private'
+DEFAULT_PROFILE_VISIBILITY = 'edx_users'
+
+# The list of fields that are shown on a shared learner profile
+SHARED_PROFILE_PUBLIC_FIELDS = [
+    'username',
+    'profile_image',
+    'country',
+    'time_zone',
+    'languages',
+    'bio',
+]
+
+# The list of fields that are shown on a private learner profile
+PRIVATE_PROFILE_PUBLIC_FIELDS = [
```

Comment:
Again private/public is confusing. These are the fields that are always shown in the profile, regardless of the selected visibility setting.